src/core/utils/compatibility.js

```diff
@@ -16,6 +16,14 @@ import { findInstallURL } from 'core/installAddon';
 import log from 'core/logger';
 
 
+// HACK: This is the ID on dev/stage/prod for the Facebook Container
+// add-on, which has a special-cased download URL supplied.
+// This should really be a special field set in the database that the front-end
+// can display, but this patch was last-minute and needed a timely turnaround.
+// See: https://github.com/mozilla/addons-server/issues/7982
+export const FACEBOOK_CONTAINER_ADDON_ID = 619864;
+export const FACEBOOK_CONTAINER_DOWNLOAD_URL = 'https://www.mozilla.org/firefox/facebookcontainer/';
```

Comment:
The issue just asked for this URL, so I wasn't sure if we wanted the UTM links in there as well; I'm not even sure that page responds to them.

But if they do and we need to tweak this patch so the UTM links are always added I can do that easily, just let me know.

Reply:
Actually, that would make more, sense, I'll update the PR with that tweak so that the UTM query params are always added.